pkg/operation/botanist/component/etcd/monitoring.go

```diff
@@ -33,16 +33,25 @@ const (
 	monitoringMetricEtcdDiskBackendCommitDurationSecondsBucket = "etcd_disk_backend_commit_duration_seconds_bucket"
 	monitoringMetricEtcdDiskWalFsyncDurationSecondsBucket      = "etcd_disk_wal_fsync_duration_seconds_bucket"
 	monitoringMetricEtcdMvccDBTotalSizeInBytes                 = "etcd_mvcc_db_total_size_in_bytes"
+	monitoringMetricEtcdMvccDBTotalSizeInUseInBytes            = "etcd_mvcc_db_total_size_in_use_in_bytes"
```

Comment:
Can you explain the difference to `etcd_mvcc_db_total_size_in_bytes`, why we need it and why it's not listed in https://github.com/gardener/etcd-druid/blob/master/docs/multinode-metrics.md?

Reply:
Got it, thanks.